react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+export default class PasswordForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { password: '' };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+
+  render() {
```

Comment:
Great job on getting this to work, I would recommend refactoring so that you don't have three separate forms in the render method, and that you conditionally render the error message.